src/Rings/mpolyquo-localizations.jl

```diff
@@ -2088,5 +2100,41 @@ function (W::MPolyDecRing)(f::MPolyLocRingElem)
   return W(forget_decoration(W)(f))
 end
 
+@attr Map function _as_affine_algebra_with_many_variables(
+    L::MPolyQuoLocRing{<:Any, <:Any, <:Any, <:Any, <:MPolyPowersOfElement}
+  )
+  inverse_name=:_0
+  R = base_ring(L)
+  f = denominators(inverted_set(L))
+  f = sort(f, lt=(x, y)->total_degree(x)>total_degree(y))
+  r = length(f)
+  A, phi, t = _add_variables_first(R, [Symbol(String(inverse_name)*"$k") for k in 1:r])
+  theta = t[1:r]
+  I = ideal(A, [phi(g) for g in gens(modulus(underlying_quotient(L)))]) + ideal(A, [one(A)-theta[k]*phi(f[k]) for k in 1:r])
+  ordering = degrevlex(gens(A)[r+1:end])
+  if r > 0 
+    ordering = lex(theta)*ordering
+  end
+  Q = MPolyQuoRing(A, I, ordering)
+  function my_fun(g)
+    a = Q(phi(lifted_numerator(g)))
+    isone(lifted_denominator(b)) && return a
+    b = Q(phi(lifted_denominator(g)))
+    success, c = divides(a, b)
+    success || error("element can not be mapped")
+    return c
+  end
```

Comment:
I had to use this complicated workaround because the usual mapping routine for homomorphisms `L -> Q` (`L` an `MPolyQuoLocRing` and `Q` an `MPolyQuo`) is using `inv` on the quotient ring `Q`. This refers to `is_invertible_with_inverse` and, I'm afraid, this is not using the block ordering provided by me. In that case, however, I did not find how to change this. Could someone look into this with me?